### ISSUE TITLE
Docker image not working as advertised

### ISSUE THREAD

**Issue by VolkerStolz** [goblint/analyzer#67]
```
$ docker pull ...
Digest: sha256:3a7487ca8f058b8aabe69c7378dc17759a3442b54077f51cf3276e4fd97aa7e6
Status: Downloaded newer image for voglerr/goblint:latest
$ docker run voglerr/goblint /analyzer/goblint --help
Fatal error: exception # opam-version    1.2.2 (58ef3b8213100953848d362f7120a30356d7f77d)
# os              linux
opam: "execvp" failed on /analyzer/goblint: No such file or directory

```

**Comment by vogler** [comment]
Thanks, I'll look into it later today. There's also a goblint package in opam now.

**Comment by vogler** [comment]
Seems like the Ubuntu version reached its EOL...
Fixed in 8d72fc27425928dc47e9ec9383ab9d2a4682e038 and eb12a5e0891464cf2c6142a8f82a94ab333be790.
It should automatically build a new image on every push now.
